README.md

```diff
@@ -9,18 +9,18 @@
  * [Configuration ](#configuration )
  * [Sample Code](#sample-code)
  * [License](#license)
- 
- 
+
+
 ## Introduction
 
-Java sample code encapsulates APIs of the HUAWEI Push Kit server. It provides many sample programs for your reference or usage.
+Java sample code that encapsulates the APIs of the HUAWEI Push Kit server. It provides many sample programs for your reference or usage.
 
 The following table describes packages of Java sample code.
 
 | Package | Description |
 | ---- | ---- |
-| examples | Sample code packages. Each package can run independently. |
-| messaging | Package where APIs of the HUAWEI Push Kit server are encapsulated. |
+| examples | Sample code classes. Each class can run independently. |
+| message | Model classes mirroring the JSON data exchanged with the HUAWEI Push Kit server APIs. |
```

Comment:
That's messaging, because it's how the directory's called. I agree about the description though

Reply:
Actually the model classes are located under "message".

Reply:
Yeah I realised after double checking my other comments

Reply:
But we would need to keep the messaging description too then